saleor/search/utils.py

```diff
@@ -0,0 +1,14 @@
+import logging
+from haystack import connections
+
+logger = logging.getLogger(__name__)
+
+
+def update_object(instance, using='default'):
```

Comment:
Do we gain anything from having a "generic" solution like this? Why not have explicit `update_product`, `update_user` etc.?